### ISSUE TITLE
Improved display of action buttons in each row

### ISSUE THREAD

**Issue by gianlucafiore** [saltcorn/saltcorn#2340]
It would be great to have the option to only show action buttons when you hover your mouse over the row

something could also be done so that buttons can be displayed when hovering over rows in a FEED, or Comment tree

**Comment by Ingenu1ty** [comment]
Yep, handling the mouse hover event would be very useful. But I suggest to do this in a **more abstract** (and therefore more functional) model: elements should have a mouse hover event (it would also be great to handle other events: keypress, right click, tap, long tap, **text selection**, etc. . ). When an event is triggered, an action is launched, which can change the state. -> This state variable should be available in "show if" formula, CSS formula, etc. This way it will be possible to implement a **convenient** responsive pattern, like in reactJS...

**Comment by glutamate** [comment]
@gianlucafiore i agree this would be nice for desktop, problem is how to implement that on mobile?

@Ingenu1ty i'll probably do something like that with dummy variables (per table and maybe global), not sure if we will be able to do it for 1.0

**Comment by Ingenu1ty** [comment]
> i'll probably do something like that with dummy variables (per table and maybe global)

Great to hear that👍

> i agree this would be nice for desktop, problem is how to implement that on mobile?

hmm... maybe use swipe for this on mobile?..(with the ability to configure: toRight, toDown, etc...)
